ajaxSample.php

```
@@ -0,0 +1,46 @@
+<?php
+//echo '====hello'; 
+//print_r($_POST);
+
+
+require_once('config/queryOperation.php');
+
+$obj = new queryOperation();
+
+$name = $_POST['name'];
+
+$result = $obj->searchData($name);
+if (!$result)
+{
+    echo 'Retrival failed' . mysql_error();
```

Log error is not handled